src/tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
+    "strict": false,
+    "strictNullChecks": false,
+    "noUnusedLocals": false,
+    "noUnusedParameters": false,
+    "noImplicitReturns": false,
+    "noImplicitAny": false,
+    "noImplicitThis": false,
+    "noFallthroughCasesInSwitch": false,
```

Comment:
I'm uncomfortable with having all these safety checks set to false. Do we really fail all of these? How many are trivially fixable? For the ones that need more in-depth fixes, can comments be left as to why they're off?

Reply:
I made tsconfig more strict